### PR TITLE
Added command line help.

### PR DESCRIPTION
**Pull request by JakeSays**

### CODE REVIEW

File: shell/main.cc [hunk 1]
```diff
@@ -66,6 +66,31 @@ void RemoveArgument(std::vector<std::string>& args, const std::string& arg) {
...
 
+/**
+ * @brief Print command line options.
+ * @return void
+ * @relation
+ * internal
+ */
+void PrintUsage()
+{
+  printf("Usage: homescreen [options] [flutter VM arguments]\n");
```
Comment (jwinarske) [inline shell/main.cc:77]: change name to `flutter-auto`

File: shell/main.cc [hunk 2]
```diff
@@ -66,6 +66,31 @@ void RemoveArgument(std::vector<std::string>& args, const std::string& arg) {
...
+  printf("   --j=<file path>               Path to a json configuration file\n");
+  printf("   --a=<integer>                 Accessibility feature flags\n");
+  printf("   --c                           Disable cursor\n");
+  printf("   --d                           Debug backend\n");
+  printf("   --f                           Full screen\n");
+  printf("   --w=<integer>                 Output width in pixels\n");
+  printf("   --h=<integer>                 Output height in pixels\n");
+  printf("   --p=<decimal>                 Pixel ratio\n");
+  printf("   --t=<string>                  Cursor theme name\n");
+  printf("   --window-type=<string>        Window type\n");
```
Comment (jwinarske) [inline shell/main.cc:89]: Only applies to AGL-Compositor.  Maybe call it `AGL Window type`

File: shell/main.cc [hunk 3]
```diff
@@ -92,6 +117,10 @@ int main(int argc, char** argv) {
   auto cl = fml::CommandLineFromArgcArgv(argc, argv);
 
   if (!cl.options().empty()) {
+    if (cl.HasOption("help")) {
+        PrintUsage();
+        return 1;
```
Comment (jwinarske) [inline shell/main.cc:122]: use `return EXIT_FAILURE;`

**Review by jwinarske — CHANGES_REQUESTED** [review]

File: shell/main.cc (hunk 3 above)
Comment (JakeSays) [reply shell/main.cc]: Done.

**Review by jwinarske — APPROVED** [review]

**Comment by jwinarske** [comment]
Thanks @JakeSays !

**Pull request merged**